src/i18n/hi/ui.ts

```diff
@@ -54,5 +54,81 @@ export default UIDictionary({
 	'cms.navTitle': 'और CMS मार्गदर्शिकाएँ', // "More CMS guides"
 	// Migration Guides vocabulary
 	'migration.navTitle': 'और माइग्रेशन मार्गदर्शिकाएँ', // "More migration guides"
-	// Recipes
+	// Recipes vocabulary
+	'recipes.navTitle': 'और अधिक रेसिपीस', // "More recipes"
```

Comment:
It should be 'अधिक' not 'और'.

Reply:
I don't know which exact lines your targeting here, if you can clarify that for me then it would be helpful